src/Pact/Native.hs

```diff
@@ -869,12 +892,16 @@ tordV = tord V.reverse
 tordL :: (Int -> [a] -> [a]) -> Integer -> [a] -> [a]
 tordL = tord reverse
 
+indexError :: Integer -> Term Name -> Int -> Eval e (Term Name)
```

Comment:
Do we need this function?

Reply:
If not, then just put `indexError` in a where clause.

Reply:
It's copied over from the last chunk of work that i did where it got used more than once. I'll remove it and do what you suggested